modules/ossm-upgrade-24-25-changes.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/upgrading-ossm.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-upgrade-24-25-changes_{context}"]
+= Upgrade changes from version 2.4 to version 2.5
+
+== Istio OpenShift Routing (IOR) default setting change
+
+The default setting for Istio OpenShift Routing (IOR) has changed. The setting is now disabled by default.
+
+You can use IOR by setting the `enabled` field to `true` in the `gateways.openshiftRoute` specification of the `ServiceMeshControlPlane` resource.
+
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+spec:
+  gateways:
+    openshiftRoute:
+      enabled: true
+----
+
+== Istio proxy concurrency configuration enhancement
+
+For consistency across deployments, Istio now configures the `concurrency` parameter based upon the CPU limit allocated to the proxy container. For example, a limit of 2500m would set the `concurrency` parameter to 3. If you set the `concurrency` parameter to a value, then Istio uses that value to configure how many threads the proxy runs instead of using the CPU limit. 
```

Comment:
```suggestion
For consistency across deployments, Istio now configures the `concurrency` parameter based upon the CPU limit allocated to the proxy container. For example, a limit of 2500m would set the `concurrency` parameter to 3. If you set the `concurrency` parameter to a value, Istio uses that value to configure how many threads the proxy runs instead of using the CPU limit. 
```

Reply:
Perhaps for another PR, do we need to describe the concurrency parameter better? (A limit of 2500m sets it to 3 doesn't make a whole lotta sense to me.)

Reply:
Made the text update and created OSSM-6041 for the second action.